CHANGELOG.md

```diff
@@ -1,16 +1,88 @@
+# 5.0.0
+
+* The minimum runtime target is now ES2015, not ES5
```

Comment:
Idea: what about using preset-env with well defined and published browserlist,
And advise users that need to support older browser to pass it thru babel again with different browserlist?

Reply:
Not familiar with preset-env. How would that work / what does it do?

Reply:
@mweststrate check https://github.com/tkrotoff/babel-preset-env-example and https://github.com/babel/website/issues/1660

To sum up, babel-preset-env automagically only includes the polyfills and babel plugins needed for a given list of targeted browsers (`browsers: ['> 1% in FR']`)

Reply:
you also have granularity of last 2 major versions of edge, chrome, safari, and so